recipes/spdlog/all/conanfile.py

```diff
@@ -41,11 +41,14 @@ def configure(self):
         if self.options.header_only:
             del self.options.shared
             del self.options.fPIC
-        elif self.settings.os == "Windows" and self.options.shared and Version(self.version) < "1.6.0":
-            raise ConanInvalidConfiguration("spdlog shared lib is not yet supported under windows")
-        if self.settings.os != "Windows" and \
-           (self.options.wchar_support or self.options.wchar_filenames):
-            raise ConanInvalidConfiguration("wchar is only supported under windows")
+        if self.settings.os == "Windows":
+            if self.options.shared and Version(self.version) < "1.6.0":
+                raise ConanInvalidConfiguration("spdlog shared lib is not yet supported under windows")
+        else:
+            if (self.options.wchar_support or self.options.wchar_filenames):
+                raise ConanInvalidConfiguration("wchar is only supported under windows")
+            if self.settings.compiler.libcxx != "libstdc++11":
+                raise ConanInvalidConfiguration("libstdc++11 required")
```

Comment:
It it was not supported, it would have been detected in previous PRs, I don't understand this modification.

Reply:
maybe the same problem where people said from paho-mqtt-c that it does not work on Windows and broke existing builds , but here it seems to turn out it's just about getting linker flags right
https://github.com/conan-io/conan-center-index/pull/4356/files#diff-cefdb38dd5cec800413f548cb236f4f8cb7fd28165ef036164d59b456553cbd4L35

(see patch, its all about a flag)

Reply:
Yes I was a bit confused with the `CMAKE_CXX_FLAGS`, sorry for the spam.